### DISCUSSION TITLE
cannot push from Github Desktop because of error message

### DISCUSSION THREAD

**Discussion by echochalet** [community/community#22338] [Other Feature Feedback, Questions, & Ideas]
<p>I’m getting the error message,<strong> “Authentication Failed.  You may not have persmission to access the repository or the repository may have been archived.  Open options and verify that you are signed in with an account that has persmission to access this repository”.</strong></p>
<p>I’m puzzled because I do have permssion and I’m signed in to the github accoun that i want to push to!  I was able to push the first time, but not after that.</p>

**Comment by Mlocik97** [comment, marked as answer]
<p>I had 2x this problem also,… solution is to log out from github desktop, remove key (revoke) from <a href="https://github.com/settings/applications" rel="nofollow noopener">https://github.com/settings/applications</a> and log in again, which will generate new key.</p>

**Comment by nadiajoyce** [comment]
<p><a class="mention" href="/echochalet">@echochalet</a> Wanted to check in here: Did the suggestion <a class="mention" href="/mlocik97">@mlocik97</a> offer work for you?</p>
